config/routes.rb

```diff
@@ -215,6 +215,28 @@
   root :to => 'home#show'
   get "podmin", to: "home#podmin"
 
+  api_version(module: "Api::V1", path: {value: "api/v1"}) do
+    match "user", to: "users#show", via: %i[get post]
+    resources :posts, only: %i[show create destroy] do
+      resources :comments, only: %i[create index destroy] do
+        post "report" => "comments#report"
+      end
+      resource :likes, only: %i[create destroy]
+    end
+    resources :conversations, only: %i[show index create destroy] do
+      delete "visibility" => "conversation_visibilities#destroy"
+      resources :messages, only: %i[index create]
+    end
+    get "streams/activity" => "streams#activity", :as => "activity_stream"
+    get "streams/stream" => "streams#multi", :as => "stream"
```

Comment:
There is no `streams/stream` node in the [streams](https://diaspora.github.io/api-documentation/routes/streams.html) API reference. Should it be `streams/main` instead?